### PR TITLE
Read only su

### PR DESCRIPTION
**Pull request by EvanBarbour3**
## Features
- Added *read only super user*

[Delivers [#128056527](https://www.pivotaltracker.com/story/show/128056527)]

> ✋

### CODE REVIEW

File: app/Http/Controllers/UsersController.php [hunk 1]
```diff
@@ -379,15 +379,58 @@ private function fetchAvailableRoles()
...
+        return [];
+    }
+
+    /**
+     * @author EB
+     * @return \Illuminate\Support\Collection
+     */
+    private function fetchSuRoles()
+    {
+        return collect([
```
Comment (wnowicki) [inline app/Http/Controllers/UsersController.php]: - I'd prefere object way

**Comment by EvanBarbour3** [comment]
https://travis-ci.org/PayBreak/basket/jobs/152997844

`[2016-08-17 15:09:33] testing.ERROR: Frontend Cache Buster: expected CB Hash Source file does not exist. [../VERSION.md] [] []`

@srleach @wnowicki proposal thoughts? Following the `VerifyCsrfToken` middleware:

```php
/**
     * Static Accessor for use in View
     *
     * @param string $filePath
     * @return string
     * @author SL
     */
    public static function cache($filePath)
    {
        if ('testing' !== app()->environment())
        {
            return $filePath . '?v=' . self::getVersionHash();
        }

        return $filePath;
    }
```

File: public/css/main.css [hunk 2]
```diff
@@ -328,3 +328,10 @@ iframe#email-prev {
 td.has-feedback i {
     padding-top: 7px;
 }
+
+/* Menu Styling */
+.scrollable-menu {
+    height: auto;
+    max-height: 70vh;
```
Comment (srleach) [inline public/css/main.css]: Wat

Comment (EvanBarbour3) [inline public/css/main.css]: Like what? Percentages were taking into consideration the container size so that wouldn't work.

**Comment by EvanBarbour3** [comment]
@wnowicki Finished off testing additions. Can you review?

**Comment by wnowicki** [comment]
what do I need to run to make it working?

**Comment by EvanBarbour3** [comment]
@wnowicki you should run `ReadOnlySuperUserSeeder`.

```
php artisan db:seed --class=ReadOnlySuperUserSeeder --env=dev
```

**Pull request merged**